video/out/opengl/video.c

```diff
@@ -2513,7 +2508,7 @@ static void pass_dither(struct gl_video *p)
             fmt = ra_find_unorm_format(p->ra, 2, 1);
             if (!fmt)
                 fmt = ra_find_float16_format(p->ra, 1);
-            if (fmt) {
+            if (fmt && fmt->num_components == 1) {
```

Comment:
What's that for?

Reply:
Oh, left-over garbage from a no-longer relevant change.